src/otx/v2/adapters/torch/lightning/anomalib/modules/callbacks/inference.py

```diff
@@ -19,34 +19,31 @@
 
 import numpy as np
 import torch
+from datumaro.components.annotation import Bbox, Label, Polygon
```

Comment:
I will update all these things in the following PR. I would like to avoid confusions between datumaro bbox and otx.api.annotation bbox. When removing all otx api, I will make clear as like did. Thank you :)